search/tweets_engine.go

```diff
@@ -67,15 +77,21 @@ func (e *ESEngine) CombineResultsToSingleHit(resultsByLang map[string]*elastic.S
 	for _, result := range resultsByLang {
 
 		var maxScore float64
+		maxEd := es.EffectiveDate{EffectiveDate: &utils.Date{time.Now()}}
+		source, _ := json.Marshal(maxEd)
 		for _, hit := range result.Hits.Hits {
-
-			if hit.Score == nil {
-				return nil, errors.Errorf("hit score is nil for hit: %s", hit.Uid)
-			}
-
 			if *hit.Score > maxScore {
 				maxScore = *hit.Score
 			}
+			var ed es.EffectiveDate
+			if err := json.Unmarshal(*hit.Source, &ed); err != nil {
```

Comment:
Consider serialization to es.Result. The Source may contain more fields that we may need.